src/guidelines/mapping.md

```diff
@@ -0,0 +1,92 @@
+
+Generally speaking, you will need to change the size of a maint tunnel on a case-by-case basis, but it's best to lean on being more spacious.
+
+Now, 2-tile-wide (and 3-tile-wide) maintenance may seem a bit overkill at first, but the usage of general obstructions such as walls, grilles, and girders and make the actual playable space end up being (and feeling) a lot smaller (and more varied, too - make sure to place a good amount of them.)
+
+Starting with such a high amount of empty spce is also important for being able to populate maintenance with the structures / loot spawns it is generally known for.
+
+### Side Rooms
+Even if your map is small, you should make sure to add lots of side-rooms in maintenance. Though they may just seem like places to put fun setpieces in at first, these locations are deceptively important.
+
+Generally, they're a great spot for players to build, socalize, and hide in, and are a core fixture of both antagonist and (sometimes) non-antagonist gameplay.
+
+Generally speaking, all of these rooms should use the **Walled Maint Door Variant Spawner** for their entrances. These provide very effective plausible deniability for antagonists and also make for more interesting round-to-round variance. Rooms that are hybrid side-room-and-maint-exit (like the medical backroom on Toast) should generally not use the walled variant, though.
+
+Additionally, it's not a good idea to populate these rooms (or otherwise random 1x1 squares in walls, or what have you) with valuable loot that can easily immediately be rushed by anyone with map knowledge.
+
+Though, points of interest in maintenance (like SS13 KiloStation's spider-gaurded improvised shotgun, or Toast's maints security post) are generally good to have as long as there's some element of intruige or friction to getting the valuables.
+
+### Random Spawners
+Random spawners are another core fixture of maintenance. They're a core part of the game's random variance and are generally what makes maintenance so interesting to explore. Make sure to place a decent amount of them, both in structure form and on-the-ground form.
+
+Though they are important, you should also *not* use random spawners for everything. If you even have them in maintenance to begin with, it's generally a good idea to have static spawns for emergency closets / fire lockers next to important locations like arrivals or dorms, pairs of water/fuel tanks, and oxygen canisters at key locations.
+
+**Using upstream random spawners over ES ones with similar loot pools is generally not advised unless you know what you are doing,** since they usually have pretty confusing and inconsistent loot pools and are pretty hard to parse while mapping.
+In any case, at least for maintenance mapping, you're almost certainly going to get more interesting variance out of an Exposed Maintenance Spawner then something like a fluff spawner or industrial fluff spawner.
```

Comment:
```suggestion
**Using upstream random spawners over ES ones with similar loot pools is generally not advised unless you know what you are doing,** since they usually have pretty confusing and inconsistent loot pools and are pretty hard to parse while mapping.
```